app/models/movie.rb

```diff
@@ -1,5 +1,11 @@
 class Movie < ApplicationRecord
-  belongs_to :genre
-  has_and_belongs_to_many :clients
+  belongs_to :genre, foreign_key: 'genre_id'
```

Comment:
Do you need to point the foreign key?
```suggestion
  belongs_to :genre
```